tests/core/gadget-config-defaults/task.yaml

```diff
@@ -167,6 +167,8 @@ execute: |
 
     echo "The configuration for core is applied"
     snap get core "service.$SERVICE.disable" | MATCH true
+    # request a document (-d) to ensure we get an integer
+    snap get -d system refresh.retain | MATCH "refresh.retain\": +5$"
```

Comment:
hm won't this fail for the rsyslog variant of the test?

Perhaps the cleanest solution would be to have a separate variant of the test, or just piggyback on ssh_common, so:
```
# request a document (-d) to ensure we get an integer
if [ "$SPREAD_VARIANT" = "ssh_common" ]; then
    # only the ssh-common gadget sets refresh retain
    snap get -d system refresh.retain | MATCH "refresh.retain\": +5$"
else
    # other variants do not, so it will not show up in the output
    snap get -d system refresh.retain | NOMATCH "refresh.retain"
fi
```

Reply:
I modified all three gadget- files so all variant can be tested here.